NetKAN/DistantObject-default.netkan

```diff
@@ -1,14 +1,14 @@
 {
     "spec_version": "v1.16",
     "identifier":   "DistantObject-default",
-    "name":         "Distant Object Enhancement Continued default config",
+    "name":         "Distant Object Enhancement /L default config",
     "abstract":     "Default planets colors",
-    "$kref":        "#/ckan/github/TheDarkBadger/DistantObject",
+    "$kref":        "#/ckan/github/net-lisias-ksp/DistantObject",
```

Comment:
:point_up:

Reply:
Yes, the DistantObject-default thingy.

I don't understand how this "filter" thingy works yet, so I think it's wiser to change this one after I get a grasp on it.

It appears to had been useful in the bast (different configs for Stock, RO, etc), but currently there's only one.

Reply:
There's no filter on that line. That's the line that needs to be updated to reference SpaceDock instead of GitHub.

Reply:
Dude, you saved my SAS. I really need to remember avoiding commits to CKAN late night. This was an automatic copy&paste that should not had happened.

There's a filter on the DOE's NetKAN file:

```
    "install": [ {
        "find"        : "DistantObject",
        "install_to"  : "GameData",
        "filter"      : "PlanetColors.cfg"
```

and a dependency on `DistantObject-config`, that is currently provided by `DistantObject-default`. And, as a matter of fact, I could not even find the `DistantObject-default.zip` file anywhere on TheDarkBadger's [repository](https://github.com/TheDarkBadger/DistantObject) - so I'm really reticent on changing this now other than the cosmetic ones.

I'm planning to replace this thing as patches using MM, but this will not happen for a couple releases - so until there, I want the `-default` to pinpoint the TheDarkBadger repo to keep things going (however they do, because I just don't understood it yet).

Later, I want to remove the `DistantObject-config` as a dependency and then freeze the `-default` one.

Reply:
`DistantObject-default` just pulls the `PlanetColors.cfg` file out of the same main ZIP file (not `DistantObject-default.zip`) and installs it, and the `filter` property prevents the main `DistantObject` from installing that file. Previously this was so that `AstronomersPack-DistantObjectEnhancement` and `DistantObject-RealSolarSystem` could install their own replacements for `PlanetColors.cfg`, but those modules are both now frozen.

Replacing this mostly-vestigial mess with ModuleManager patches is a good idea. :+1: More maintainable, easier installation for non-CKAN users. In the meantime, though, it's probably wise to make the `$kref` properties the same, since they're supposed to be pulling from the same ZIP file.

Reply:
Now I understand. I will fix them and commit them for review ASAP.